packages/camera/camera_avfoundation/ios/Classes/CameraPlugin.m

```diff
@@ -299,6 +299,14 @@ - (void)handleCreateMethodCall:(FlutterMethodCall *)call
   });
 }
 
+// Returns integer value if provided and positive, or nil if not specified or negative.
```

Comment:
Could you add more context to this comment on why it exists? It's non-obvious to me why converting negative numbers to `nil` would be desirable.

Reply:
```objc
// Returns number value if provided and positive, or nil.
// Used to parse values like framerates and bitrates, that are positive by nature.
// nil allows to ignore unsupported values.
NSNumber *toPositiveOrNil(id number) {
```

Ok?

Reply:
> ```objc
> // Used to parse values like framerates and bitrates, that are positive by nature.
> // nil allows to ignore unsupported values.
> ```

Why would we want to silently ignore incorrect values, rather than treat them as argument errors?